sbom-utility-scripts/Dockerfile

```diff
@@ -7,5 +7,7 @@ COPY scripts/merge-cachi2-sboms-script/merge_cachi2_sboms.py /scripts
 COPY scripts/create_purl_sbom.py /scripts
 COPY scripts/base-images-sbom-script/app/base_images_sbom_script.py /scripts
 COPY scripts/base-images-sbom-script/app/requirements.txt /scripts
+COPY scripts/index-image-sbom-script/index_image_sbom_script.py /scripts
 
+# Same requirements for base-images-sbom-script and index-image-sbom-script
 RUN pip3 install -r requirements.txt
```

Comment:
> very nasty surprises

For example, the index-image requirements.txt currently specifies a higher version of `packageurl-python` than the base-images requirements.txt. That means we're unit testing with a different version that what we'll be running in prod, which is a bit scary

Reply:
Thanks for pointing that out. Changed according to your proposal and unified the package versions of the 2 scripts (pip installation fails otherwise).